modules/lb-vshn-hiera/main.tf

```
@@ -0,0 +1,22 @@
+locals {
+  instance_fqdns = formatlist("%s.${var.node_name_suffix}", var.lb_names)
+
+  lb_count = length(var.lb_names)
+
+  content = templatefile(
+    "${path.module}/templates/hieradata.yaml.tmpl",
+    {
+      "cluster_id"   = var.cluster_id
+      "api_vip"      = cidrhost(var.api_vip_network, 0)
+      "router_vip"   = cidrhost(var.router_vip_network, 0)
+      "api_secret"   = var.lb_cloudscale_api_secret
+      "internal_vip" = cidrhost(var.privnet_cidr, 100)
+      "nat_vip"      = cidrhost(var.nat_vip_network, 0)
```

Same here, also as mentioned in the variables file, privnet_cidr doesn't make sense for Exoscale:

```suggestion
      "internal_vip" = var.internal_vip
      "nat_vip"      = var.nat_vip
```